components/announcement_bar/renewal_link/renewal_link.tsx

```diff
@@ -16,6 +16,7 @@ import {
 } from 'utils/constants';
 
 import NoInternetConnection from '../no_internet_connection/no_internet_connection';
+
```

Comment:
changes on this file were just remove unnecessary type React.FC from component, and remove unnecessary fragment surrounding component content.